App.js

```diff
@@ -8,7 +8,6 @@ import { Ionicons } from '@expo/vector-icons';
 
 import firebase from 'firebase';
```

Comment:
Can we delete this import as it's unused?

Can we also delete "import { Asset } from 'expo-asset';" as it is also unused?